App/App.tsx

```diff
@@ -107,3 +111,5 @@ export function App(): React.ReactElement {
 		</ErrorContextProvider>
 	);
 }
+
+// TODO bug will be encounter until all the old translation is replaced
```

Comment:
So running the current code will be bugged?

Reply:
This one belongs to a new update (at that moment). There was a package moved out to a dependency. Therefore, it will cause bug. But this one was meant to. In order to fix it. I have to fix from the dependency. There is nothing I can do about it on this PR.

(I would like to discuss with you about that. How we should handler it since last time didn't go well right?)

Reply:
If you open 'About' page, you'll see 2 missing strings. Those are `about_box_per_day` and `about_box_footnote`. Those were components which had been moved out since a few previous update. Therefore, in the scope of this PR, I have no choice but to add that note.